Firebase/Messaging/CHANGELOG.md

```diff
@@ -0,0 +1,85 @@
+# 2018-02-06 -- v2.1.0
+- Adding a new property autoInitEnabled to enable and disable FCM token auto generation.
+- Fixed an issue of un-successful delivery of notification message when changing language settings.
+
+# 2017-09-26 -- v2.0.5
+- Added swizzling of additional UNUserNotificationCenterDelegate method, for
+  more accurate Analytics logging.
+- Fixed a swizzling issue with un-implemented UNUserNotificationCenterDelegate
+  methods.
+
+# 2017-09-26 -- v2.0.4
+- Fixed an issue where the FCM token was not associating correctly with an APNs
+  device token, depending on when the APNs device token was made available.
+- Fixed an issue where FCM tokens for different Sender IDs were not associating
+  correctly with an APNs device token.
+- Fixed an issue that was preventing the FCM direct channel from being
+  established on the first start after 24 hours of being opened.
+- Clarified a log message about method swizzling being enabled.
+
+# 2017-09-13 -- v2.0.3
+- Moved to safer use of NSAsserts, instead of lower-level __builtin_trap()
```

Comment:
`__builtin_trap()` should be escaped with backticks so the double underscore isn't parsed incorrectly as markdown italics.